src/ui/app/jsx/repair-form.jsx

```diff
@@ -360,11 +380,48 @@ const repairForm = CreateReactClass({
     return uuid;
   },
 
+  _onForce(e){
+    this.setState({
+      force: "true",
+      addRepairResultMsg: null,
+      showModal: false }, () => this._onAdd(e));
+  },
+
+  _onClose(e){
+    this.setState({
+      force: "false",
+      addRepairResultMsg: null,
+      showModal: false
+    });
+  },
+
   render: function() {
 
     let addMsg = null;
     if(this.state.addRepairResultMsg) {
-      addMsg = <div className="alert alert-danger" role="alert">{this.state.addRepairResultMsg}</div>
+      if(this.state.addRepairResultStatus != 409) {
+        addMsg = <div className="alert alert-danger" role="alert">{this.state.addRepairResultMsg}</div>
+      }
+      else {
+        addMsg = (
+                <span>
+                <Modal show={this.state.showModal} onHide={this._onClose}>
+                  <Modal.Header closeButton>
+                    <Modal.Title>Scheduling conflict detected</Modal.Title>
+                  </Modal.Header>
+                  <Modal.Body>
+                    <p>{this.state.addRepairResultMsg}</p>
+                    <p>It is not reccommended to create overlapping repair schedules.</p>
+                    <p>For Cassandra 4.0 and later, you can force creating this schedule by clicking the "Force" button below.</p>
+                  </Modal.Body>
```

Comment:
@adejanovski I looked back at the ticket again and there was a comment about making the error returned from the REST call be the same as the message in the pop-up here. Should I move all this text back into the Java code where the 409 Conflict is created? Or is it ok the way it is.

Currently, the REST call will have a 409 with text:
`A repair schedule already exists for cluster \"%s\", keyspace \"%s\", and column families: %s`
and none of the extra text I've added to the pop-up here.

Reply:
I'm happy with keeping it the way it is 👍 It gives better formatting capabilities than just displaying the returned message (which shouldn't be formatted in html anyway).